src/app/settings/reducer.ts

```diff
@@ -3,79 +3,13 @@ import * as actions from './actions';
+import { Settings as DimApiSettings, defaultSettings } from '@destinyitemmanager/dim-api-types';
 
 export type CharacterOrder = 'mostRecent' | 'mostRecentReverse' | 'fixed' | 'custom';
 
-export interface Settings {
-  /** Show full details in item popup */
-  readonly itemDetails: boolean;
-  /** Show item quality percentages */
-  readonly itemQuality: boolean;
-  /** Show new items with an overlay */
-  readonly showNewItems: boolean;
-  /** Show item reviews */
-  readonly showReviews: boolean;
-  /** Can we post identifying information to DTR? */
-  readonly allowIdPostToDtr: boolean;
-  /** Sort characters (mostRecent, mostRecentReverse, fixed) */
-  readonly characterOrder: CharacterOrder;
-  /**
-   * Sort items in buckets (primaryStat, rarityThenPrimary, quality).
-   * This used to let you set a preset but now it's always "custom"
-   * unless loaded from an older settings.
-   */
-  readonly itemSort: string;
-  readonly itemSortOrderCustom: string[];
-  /** How many columns to display character buckets */
-  readonly charCol: number;
-  /** How many columns to display character buckets on Mobile */
-  readonly charColMobile: number;
-  /** How big in pixels to draw items - start smaller for iPad */
-  readonly itemSize: number;
-  /** Which categories or buckets should be collapsed? */
-  readonly collapsedSections: { [key: string]: boolean };
-  /** Hide triumphs once they're completed */
-  readonly completedRecordsHidden: boolean;
-  /** Show triumphs the manifest recommends be redacted */
-  readonly redactedRecordsRevealed: boolean;
-  /** Whether to keep one slot per item type open (D1 only) */
-  readonly farmingMakeRoomForItems: boolean;
-  /** Destiny 2 platform selection for ratings + reviews */
-  readonly reviewsPlatformSelectionV2: DtrReviewPlatform;
-  /** Destiny 2 play mode selection for ratings + reviews - see DestinyActivityModeType for values */
-  readonly reviewsModeSelection: DtrD2ActivityModes;
-
-  /** Hide completed Destiny 1 records */
-  readonly hideCompletedRecords: boolean;
-
-  /** Custom character sort - across all accounts and characters! */
-  readonly customCharacterSort: string[];
-
-  /** The last direction the infusion fuel finder was set to. */
-  readonly infusionDirection: InfuseDirection;
-
-  /** Whether the item picker should equip or store. */
-  readonly itemPickerEquip: boolean;
-
-  /** The user's preferred language. */
-  readonly language: string;
-
-  /** Colorblind modes. */
-  readonly colorA11y: string;
-
-  /**
-   * External source for wish lists.
-   * Expected to be a valid URL.
-   * initialState should hold the current location of a reasonably-useful collection of rolls.
-   * Set to empty string to not use wishListSource.
-   */
-  readonly wishListSource: string;
-
+export interface Settings extends DimApiSettings {
   /** list of stat hashes of interest, keyed by class enum */
   readonly customTotalStatsByClass: KeyedStatHashLists;
```

Comment:
This'll get folded into DimApiSettings eventually - it just wasn't there when I first copied it over.